tests/test_trans_builder.py

```diff
@@ -1,10 +1,12 @@
 """Test Transactin Builder class."""
```

Comment:
Did you add any unit test to test the new functionality (e.g. passing in the new params into with_line or exempt_line)?

Reply:
These scenarios are partially covered in existing test cases (modified them to cover functionality).
Added 2 new test cases for covering all scenarios.